src/Shared/Configuration/Monai.Deploy.WorkflowManager.Configuration.csproj

```diff
@@ -48,6 +48,10 @@
     <AdditionalFiles Include="..\..\.sonarlint\project-monai_monai-deploy-workflow-manager\CSharp\SonarLint.xml" Link="SonarLint.xml" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\WorkflowManager\Contracts\Monai.Deploy.WorkflowManager.Contracts.csproj" />
```

Comment:
Since this is a shared project, can we try and keep this separate from the workflow manager contracts?

Reply:
Might mean some duplication but probably better to be as separate as possible

Reply:
just tried it out and appears I've just mistakenly added the project reference and not removed it again when I've moved code out of the project.